### PR TITLE
feat():implementation of  workcompletion graph completed

### PR DESCRIPTION
**Pull request by Kanishk6677**
Screenshot from 2021-06-08 20-30-48
Signed-off-by: Kanishk6677 <kanishkraj6677@gmail.com>

### Functionality:
Implemented Work Completion Graph for a given particular task

### Solution:
Used ng circle session charts to demonstrate the work done by passing the data of workdone

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
ng serve
firebase emulators: start

### CODE REVIEW

**Comment by viv9k** [comment]
Is there any chart in Google chart?

If so better to use Google chart.

**Comment by viv9k** [comment]
Is there any chart in Google chart?

If so better to use Google chart.

**Comment by Kanishk6677** [comment]
I was facing some problems in Google charts. Their pie's data is fetched
with little bit complexities

File: src/app/app.module.ts [hunk 1]
```diff
@@ -96,6 +100,19 @@ import { UserVerificationComponent } from './user-verification/user-verification
...
+    
+      radius: 15,
+      outerStrokeWidth: 5,
+      innerStrokeWidth: 10,
+      outerStrokeColor: "#78C000",
+      innerStrokeColor: "#C7E596",
+      animationDuration: 300,
+      showSubtitle:false,
+      titleFontSize:'8',
+      titleColor:"#228B22"
```
Comment (viv9k) [inline src/app/app.module.ts:113]: Please try moving these information to component

**Review by viv9k — CHANGES_REQUESTED** [review]
Let's connect once to review this

**Comment by viv9k** [comment]
Can we take quick action on this.

**Comment by Kanishk6677** [comment]
Sure, will do that. This week I had stuck with some issues. I will complete
it as soon as possible.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**